library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
Having modules live in the root of the repo reduce re-usability, since they will not be in the lookup path if called from a playbook that exists outside of openshift-ansible.

If there is a module that needs to be used by more than one role, it should be handled by role dependencies.